google-cloud-pubsub/lib/google/cloud/pubsub/async_publisher.rb

```diff
@@ -79,6 +82,15 @@ def initialize topic_name, service, max_bytes: 10000000,
           @publish_threads  = (threads[:publish] || 4).to_i
           @callback_threads = (threads[:callback] || 8).to_i
 
+          @published_at = nil
+          @publish_thread_pool  = Concurrent::ThreadPoolExecutor.new \
+            max_threads: @publish_threads
+          @callback_thread_pool = Concurrent::ThreadPoolExecutor.new \
+            max_threads: @callback_threads
+          @thread ||= Thread.new { run_background }
+
+          @batches = {}
+
```

Comment:
Initialize `@ordered` to false to prevent uninitialized instance variable warnings

Reply:
Added.